src/View/Requirements_Backend.php

```diff
@@ -494,9 +494,47 @@ protected function getAllJavascript()
     public function customScript($script, $uniquenessID = null)
     {
         if ($uniquenessID) {
-            $this->customScript[$uniquenessID] = $script;
+            $this->customScript[$uniquenessID]['content'] = $script;
         } else {
-            $this->customScript[] = $script;
+            $this->customScript[]['content'] = $script;
+        }
+    }
+
+    /**
+     * Register the given Javascript code into the list of requirements with optional tag
+     * attributes.
+     *
+     * @param string     $script       The script content as a string (without enclosing `<script>` tag)
+     * @param array $options List of options. Available options include:
+     * - 'type' : Specifies the type of script
+     * - 'crossorigin' : Cross-origin policy for the resource
+     * @param string|int $uniquenessID A unique ID that ensures a piece of code is only added once
+     */
+    public function customScriptWithAttributes($script, $options = [], $uniquenessID = null)
```

Comment:
```suggestion
    public function customScriptWithAttributes(string $script, array $attributes = [], string|int|null $uniquenessID = null)
```

Reply:
Yep, I can update this, similar to below I wanted to keep the requirements class consistent. Would it be of use to make an issue to update the entirety of requirements to follow PHP8 standards?

Reply:
No, consistently strong typing everything in Silverstripe is a huge job and a not a concern right now.  Currently we're just ensuring that any new API added is strongly typed